Doppler.Currency/Startup.cs

```diff
@@ -63,6 +63,35 @@ public void ConfigureServices(IServiceCollection services)
                     Version = "v1",
                     Description = "API for Doppler Currency"
                 });
+
+                var security = new Dictionary<string, IEnumerable<string>>
+                {
+                    {"Bearer", new string[] { }},
+                };
+
+                c.AddSecurityDefinition("Bearer", new OpenApiSecurityScheme
+                {
+                    Description = "JWT Authorization header using the Bearer scheme (Example: 'Bearer 12345abcdef')",
+                    Name = "Authorization",
+                    In = ParameterLocation.Header,
+                    Type = SecuritySchemeType.ApiKey,
+                    Scheme = "Bearer"
+                });
+
+                c.AddSecurityRequirement(new OpenApiSecurityRequirement
+                {
+                    {
+                        new OpenApiSecurityScheme
+                        {
+                            Reference = new OpenApiReference
+                            {
+                                Type = ReferenceType.SecurityScheme,
+                                Id = "Bearer"
+                            }
+                        },
+                        Array.Empty<string>()
```

Comment:
In one place you are using `new string[] { }` and in another place you are using `Array.Empty<string>()` to create a empty string array. Is it possible choose one option?

Reply:
I used the Array.Empty<string>() option for both cases